2333/1/koans/triangle.rb

```diff
@@ -0,0 +1,39 @@
+# Triangle Project Code.
+# :reek:UncommunicativeParameterName
+# :reek:UncommunicativeVariableName
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+# :reek:TooManyStatements
+# :reek:FeatureEnvy
+def triangle_raise_error(side_a, side_b, side_c)
```

Comment:
Hi. I'm a mentor from Rubizza. Alena asked to help a bit with reviews, so here am I :)

Everything looks pretty cool! I liked that you moved the validation logic out for another method. One thing though which disturb me a bit is triangle_raise_error method name.
Let's rename to something more cheerful, like validate_sides. What do you think?